news/2025/2025-10-30-fall-time-mania-mapping-contest.md

```diff
@@ -0,0 +1,83 @@
+
+| Event | Dates |
+| --: | :-- |
+| Beatmapping (3.5 weeks) | October 30th - November 24th (16:00 UTC) |
+| Judging (2 weeks) | November 24th - December 8th (16:00 UTC) |
+| Results (1 week) | December 15th or sooner depending on processing |
+
+Results will be made public through the Discord server as well as through a conclusion news post.
+
+## Prizes
+
+This contest comes packed with prizes so do not fret! Your efforts will be rewarded!
+
+We've prepared some lovely prizes to reward those who impress our judges the most! We also have a category called **Judge's Pick** which will feature the entry that the judges deemed worthy of an extra prize!
+
+| Placement | Prizes |  |
+| :-: | :-- | :-- |
+| 1st | 3 months of osu!supporter, **[contest points](/wiki/Contests/Contest_points)**, **profile badge** | ![Badge](https://i.gyazo.com/154b75d167e0d4c6a60b4606df51b889.png) |
+| 2nd | 2 months of osu!supporter |  |
+| 3rd | 1 month of osu!supporter |  |
+| Judge's Pick | 3 months of osu!supporter |  |
+
+The amount of contest points awarded will be determined by the number of mappers participating in the contest, as detailed in the [contest points key](/wiki/Contests/Contest_points#points-key).
+
+## Judging Criteria
+
+For more details regarding the criteria, please take a look at the [forum post]() as it details it further.
+
+| Category | Details |
+| :-: | :-- |
+| Musical Representation (10) | How effectively the beatmap's rhythms, object placements and pattern choices fit the music. |
+| Creativity (10) | How inventive the beatmap's structure/design choices are. |
+| Gameplay (10) | How engaging the beatmap is to play as well as taking into account if your choice in patterns, structure and design affect the experience overall. |
+| Gimmick (5) | Your map's final object count should be a multiple of either 9,10,11 or 12 to signify the months which autumn falls under. |
+
+## Rules
+
+- **Only use songs from our [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists).** You can choose whatever artist or song you'd like as long as it's on our FA listing, if your submission is not part of the FA listing, it'll be disqualified.
+- **Submissions must be in `.osz` format**.
+- **Submissions must only be of the osu!mania game mode**. Any other game mode will not be accepted.
+- **Submissions must adhere to the [ranking criteria](/wiki/Ranking_Criteria) as well as the [osu!mania ranking criteria](wiki/Ranking_criteria/osu!mania)**. The winner will have a guaranteed BN for their map.
+- **Submissions must not be publicly revealed until results are out, this includes sharing the map and/or submitting it through the editor**.
+- Cut versions are allowed only if it comes from our Featured Artist listing, the cut makes sense and it’s appropriately titled as such in the map’s title `(Cut ver.)`.
```

Comment:
Incorrect apostrophes, plus this is technically not your "FA Listing", but rather the game's. Plus oxford comma.
```suggestion
- Cut versions are allowed only if it comes from the Featured Artist listing, the cut makes sense, and it's appropriately titled as such in the map's title `(Cut ver.)`.
```

Side note, but the rule is somewhat confusing. Particularly the second part of it. What do you mean by the "cut making sense" if the first part simply allowed cuts directly from the FA listing? Does it mean that users can make their own cuts so long songs come from the listing? Does it mean that only *some* cuts in the listing are allowed? If so, which of the cuts in the listing are allowed? It's somewhat up to interpretation, would make it clearer.

Reply:
Will modify this.